examples/nextjs-route-handlers-upload/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
have you thought about moving prettier to the root of the project?

that way we dont have to manage this in every single repo, it would be a bit cleaner and less to worry about in each example and inherit from the root of the project

Reply:
I noticed there is some inconsistency between prettier configs in other projects. Like svelte, vue, etc
I will take the commons from all projects using prettier and put that in the root, and other settings like `prettier-plugin-svelte` will keep them in the individual files

Does that work?

Reply:
if it makes sense, yeah go for it